### ISSUE TITLE
Establish support case tracking process

### ISSUE THREAD

**Issue by cbeams** [bisq-network/projects#5]
_/cc @bisq-network/support-agents (so everyone is aware of this work in progress)_

## Delivery criteria
 - [ ] Wiki documentation of the [case tracking process](https://bisq.wiki/Support_Agent#Case_tracking) is complete
 - [ ] All support agents understand how to track their cases and are actually doing so
 - [ ] Script(s) exist that process closed support case issues and produce the simple metrics discussed in the Q1 2020 Update call (#5), including:
   - Support cases per cycle
   - Support cases per trade
   - Mean time to response
   - Mean time to resolution
   - Support cases per known issue / critical bug (this one is probably not scriptable per se, but a process can be put in place to make getting the information feasible)

## Tasks
 - [x] Decide to use bisq-network/support repo issues for tracking
 - [x] Announce this decision in the support team kickoff call (#19)
 - [x] Create support repo [Case Tracking project board](https://github.com/bisq-network/support/projects/2)
 - [x] Automate adding newly created issues to the Case Tracking Board (#6)
 - [x] Give `@bisq-network/support-agents` Triage access to the support repo
 - [x] Experiment with issue title formats and encrypting sensitive data contents with Keybase (#7)
 - [x] Define metadata fields for programmatic parsing of closed support cases (bisq-network/support#336)
 - [x] Create new `support case` issue label in the support repository
 - [ ] Create support case issue template containing the metadata fields mentioned above, and YAML front matter that labels issues as 'support case' (bisq-network/support#336)
 - [ ] Figure out a dead-simple way that all support agents can quickly convert their local time in keybase to UTC/zulu time to copy and paste into the date metadata fields of each new issue
 - [ ] Create a short screencast showing how to follow the case tracking process (@cbeams)
 - [ ] Finish documenting the process for tracking support cases on the Support Agent wiki page
   - [ ] Include a link to the screencast
 - [ ] Get bisq-network/admin#18 (wiki notifications) implemented
 - [ ] Send a message to `@bisq-network/support-agents` linking to the wiki and asking everybody to read it and take action, and to please :+1: this message to indicate you got it and will do. Mention / link to the screencast. Ask to respond with any questions or feedback, and to make edits to the wiki as appropriate (#18 needs to be done first!). Also ask agents to #reference this issue when they create their first case tracking issue, so we can easily realize when the task immediately following below is finished.
 - [ ] Leave this issue open until all current `@l1-support-agents` and `@intern-support-agents` have created at least one support case issue correctly
   - [ ] `@bisq-knight`
   - [ ] `@leo816`
   - [ ] `@huey735`
   - [ ] `@mwithm`
   - [ ] `@bayernator`
   - [ ] `@wiz` (?)
   - [ ] `@m52go` (?)
 - [ ] Write script / application that queries the GitHub Issues API to produce the metrics listed above in delivery criteria
 - [ ] Make it a duty of the support team lead role to run that script and report on those metrics every cycle.

**Comment by cbeams** [comment]
Closing as aborted. Per https://github.com/orgs/bisq-network/teams/dao/discussions/13, I'll be stepping down from the role of Support Team Lead (bisq-network/roles#102), and rather than leave this open or try to hand it off to someone else (e.g. the new team lead), it's probably better to call the shot that this has, in the end, not been important enough to get done, and to just drop it. We are unable to really measure support activities without this kind of tracking, but user support is no longer a serious pain point for the project; it appears to be going pretty smoothly and perhaps the current process, e.g. coverage calendar and escalation process is "good enough" for the time being. The one thing that's really missing without this kind of tracking in place is getting good data about which critical bugs and issues are causing issues to land in support, but in practice, we know what most of these are anyway; it's just a matter of actually fixing / addressing them.
